Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,12 @@
 function sum(elements) {
+        let total = 0;
+    for (let item of elements) {
+        if (typeof item === 'number') {
+            total += item; 
+        }
+    }
+    return total;
+
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
sum([Infinity, 1]);
sum([Infinity, Infinity]);
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```

Reply:
I updated the code, thanks